### ISSUE TITLE
Can anyone read the code?

### ISSUE THREAD

**Issue by k1ln** [nodejitsu/node-http-proxy#941]
Hi! No Biggie... But can you really see through the f..... f..... you wrote there in the code?????

I tried for 3 hours and don't understand balls.....

Cheers

**Comment by k1ln** [comment]
For example: can i readout the html from the web here?:
 proxyReq.on('response', function(proxyRes) {
      if(server) { server.emit('proxyRes', proxyRes, req, res); }
      for(var i=0; i < web_o.length; i++) {
        if(web_o[i](req, res, proxyRes, options)) { break; }
      }

      // Allow us to listen when the proxy has completed
      proxyRes.on('end', function () {
        server.emit('end', req, res, proxyRes);
      });

If i would like to read out what the page gets for example. because i can't get the data string.

**Comment by tarkeshwars** [comment]
No. You can't readout the html of the page. The response is getting piped. So it's not like the page gets loaded and then retransmitted. If you want to parse/modify the html that you are getting, look into harmon plugin that works with node-http-proxy.

**Comment by acanimal** [comment]
Hi, I don't know if it is exactly the same but some time ago I require to read the response data. See http://www.acuriousanimal.com/2015/08/31/how-to-read-from-a-writable-stream-httpserverresponse-in-node.html

**Comment by k1ln** [comment]
Ok. Thanks a lot. 

The Problem is when i tried to access 'http://www.handelsblatt.com' it says: Forbidden can't access "/" on this server and i wanted to know if that is what was transmitted. Same with redirects if not enabled no redirect-code is shown. Did you get that error already.

**Comment by Mythli** [comment]
If anyone is interested into that, its cuz handelsblatt does not know what to do with the proxied request, after all it does not know what to do with your transmitted hostname localhost.

Change the header accordingly:

 proxyReq.setHeader('host', 'www.handelsblatt.com');
